lib/src/foundation/bloc.dart

```diff
@@ -29,13 +29,18 @@ class Bloc {
   }
 
   // The different streams send the featureId that must display/complete.
+  // The three types of actions are open, complete, and dismiss and
+  // they are ordered like that, like everywhere else as well.
 
-  final StreamController<String> _dismissController =
-      StreamController.broadcast();
+  // Open
 
-  Stream<String> get outDismiss => _dismissController.stream;
+  final StreamController<String> _openController = StreamController.broadcast();
```

Comment:
I organized this a bit and called this "open" instead of "start" like everywhere else. I also applied this consistency to the other occurences of the stream.

Git detected the reordering pretty fine. It is always a bit annoying that git blame is messed up by changes like this, but I hope that this is okay for you.